ocp_resources/machine_config_pool.py

```diff
@@ -7,3 +7,30 @@ class MachineConfigPool(Resource):
     class Status(Resource.Status):
         UPDATED = "Updated"
         UPDATING = "Updating"
+
+    def __init__(
+        self,
+        name,
+        labels,
+        machine_config_selector_matches,
+        configuration=None,
+    ):
+        super().__init__(name=name)
+        self.resource_labels = labels
+        self.configuration = configuration
+        self.machine_config_selector_matches = machine_config_selector_matches
+
+    def to_dict(self):
+        super().to_dict()
+        if self.resource_labels:
+            self.res["metadata"] = []
```

Comment:
can be done in one line with list comprehension

Reply:
Done